ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml

```diff
@@ -1,29 +1,31 @@
 apiVersion: nmstate.io/v1
 kind: NodeNetworkConfigurationPolicy
 metadata:
- name: ceph-public-net-shim-worker-node
- namespace: openshift-storage
+  name: ceph-public-net-shim-worker-node
+  namespace: openshift-storage
```

Comment:
please delete the space.

Reply:
I think it somehow messed here in how github present it. This is how it looks in the editor:
![image](https://github.com/user-attachments/assets/6e73adb1-5cea-492e-9244-4b9a4cf62bb6)

Reply:
The indentation of the file was just one space and the editor fixed it to have 2 spaces which is our regular formatting of yaml